repositories/src/test/scala/cromwell/pipeline/datastorage/dao/repository/utils/GeneratorUtils.scala

```diff
@@ -0,0 +1,123 @@
+package cromwell.pipeline.datastorage.dao.repository.utils
+
+import java.nio.file.{ Path, Paths }
+
+import cats.implicits._
+import cromwell.pipeline.datastorage.dto._
+import cromwell.pipeline.datastorage.dto.auth.{ SignInRequest, SignUpRequest }
+import cromwell.pipeline.datastorage.dto.user.{ PasswordUpdateRequest, UserUpdateRequest }
+import cromwell.pipeline.model.validator.Enable
+import cromwell.pipeline.model.wrapper._
+import org.scalacheck.Gen
+
+import scala.util.Random
+
+object GeneratorUtils {
+  private def stringGen(n: Int): Gen[String] = Gen.listOfN(n, Gen.alphaLowerChar).map(_.mkString)
+
+  private lazy val booleanGen: Gen[Boolean] = Gen.oneOf(List(false, true))
+
+  private lazy val projectIdGen: Gen[ProjectId] = Gen.uuid.map(id => ProjectId(id.toString))
+
+  private lazy val repositoryGen: Gen[Repository] = stringGen(10).map(Repository(_))
+
+  private lazy val emailGen: Gen[UserEmail] = for {
+    name <- stringGen(10)
+    mail <- Gen.oneOf(Mail.values.toSeq)
+    domain <- Gen.oneOf(Domain.values.toSeq)
+  } yield UserEmail(s"$name@$mail.$domain", Enable.Unsafe)
+
+  private lazy val nameGen: Gen[Name] = for {
+    name <- stringGen(6)
+  } yield Name(name, Enable.Unsafe)
+
+  private lazy val pathGen: Gen[Path] = Gen
+    .choose(1, 5)
+    .flatMap(length => Gen.listOfN(length, stringGen(5)).map(_.mkString("/")).map(path => Paths.get(path)))
+
+  private lazy val pipelineVersionGen: Gen[PipelineVersion] =
+    new PipelineVersion(VersionValue.random, VersionValue.random, VersionValue.random)
```

Comment:
No need in implicit conversions nor `VersionValue.random` method at all.
```suggestion
  private lazy val versionValueGen: Gen[VersionValue] = Gen.posNum[Int].map(VersionValue(_, Enable.Unsafe))

  private lazy val pipelineVersionGen: Gen[PipelineVersion] =
    for {
      major <- versionValueGen
      minor <- versionValueGen
      revision <- versionValueGen
    } yield PipelineVersion(major, minor, revision)
```

Reply:
Fixed, thanks